sources/logdog/conf/metal-k8s.conf

```diff
@@ -0,0 +1 @@
+exec kube-status systemctl status kube* -l --no-pager
```

Comment:
Is this different from the other k8s variants?

Reply:
The `aws` file differs, but we could probably unify the `vmware` and `metal` files since they're the same, and both are "off-AWS" variants.